diff -up pesign-0.112/Make.defaults.omv~ pesign-0.112/Make.defaults
--- pesign-0.112/Make.defaults.omv~	2018-10-03 14:24:31.939217933 +0200
+++ pesign-0.112/Make.defaults	2018-10-03 14:24:36.761238516 +0200
@@ -49,7 +49,7 @@ clang_ccldflags =
 gcc_ccldflags = -fno-merge-constants \
 	-Wl,--fatal-warnings,--no-allow-shlib-undefined,--default-symver \
 	-Wl,-O2 -Wl,--no-undefined-version -Wl,-z,relro,-z,now \
-	--Wl,--no-add-needed,--no-copy-dt-needed-entries,--as-needed
+	-Wl,--no-add-needed,--no-copy-dt-needed-entries,--as-needed
 ccldflags = $(cflags) $(CCLDFLAGS) $(LDFLAGS) \
 	$(if $(filter $(CCLD),clang),$(clang_ccldflags),) \
 	$(if $(filter $(CCLD),gcc),$(gcc_ccldflags),) \
